src/components/PostList.tsx

```diff
@@ -72,7 +72,7 @@ const PostList = ({ feedType }: PostListProps): JSX.Element => {
             })}
         </>
       ) : (
-        "Empty Feed!"
+        "Empty FeedNavigation!"
```

Comment:
Shouldn't this just stay as  "Empty Feed"?

Reply:
wow webstorm is ultra good at refactoring